docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 api:
-  image: 'docker.io/c2corg/v6_api:markdown-cooker'
+  image: 'docker.io/c2corg/v6_api:suglify-dep'
```

Comment:
shouldn't this be reset to dev-environment or something similar?

Reply:
yes :)